demos/multi_material/compositional_buoyancy/compositional_buoyancy.py

```diff
@@ -131,25 +133,21 @@
 
 # fig, axes = plt.subplots()
 # axes.set_aspect("equal")
-# contours = tricontourf(psi, levels=linspace(0, 1, 11), axes=axes, cmap="PiYG")
+# contours = tricontourf(psi, levels=linspace(0.0, 1.0, 11), axes=axes, cmap="PiYG")
 # tricontour(psi, axes=axes, levels=[0.5])
-# fig.colorbar(contours, label="Conservative level-set")
+# fig.colorbar(contours, label="Conservative level set")
 # -
 
-# We next define materials present in the simulation using the `Material` class. Here,
-# the problem is non-dimensionalised and can be described by the product of the
-# expressions for the Rayleigh and buoyancy numbers, RaB, which is also referred to as
-# compositional Rayleigh number. Therefore, we provide a value for thermal and
-# compositional Rayleigh numbers to define our approximation. Material fields, such as
-# RaB, are created using the `field_interface` function, which generates a unique field
-# over the numerical domain based on the level-set field(s) and values or expressions
-# associated with each material. At the interface between two materials, the transition
-# between values or expressions can be represented as sharp or diffuse, with the latter
-# using averaging schemes, such as arithmetic, geometric, and harmonic means.
-
+# We next define the material fields and instantiate the approximation. Here, the system
+# of equations is non-dimensional and only includes compositional buoyancy under the
+# Boussinesq approximation. Moreover, physical parameters are constant through space
+# apart from density. As a result, the system is fully defined by the compositional
+# Rayleigh number. We use the `material_field` function to define its value throughout
+# the domain (including the shape of the material interface transition) and provide it
+# to our approximation.
 
 # +
-Ra = 0  # Thermal Rayleigh number
+Ra = 0.0  # Thermal Rayleigh number
```

Comment:
Will be nice when we can drop this!

Reply:
(It has only been a year since I said that for the first time.)